corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -55,10 +55,14 @@ hqDefine("app_manager/js/details/screen", function () {
         self.containsSearchConfiguration = options.containsSearchConfiguration;
         self.containsCustomXMLConfiguration = options.containsCustomXMLConfiguration;
         self.allowsTabs = options.allowsTabs;
+        self.caseTileTemplateOptions = options.caseTileTemplateOptions;
+        self.caseTileTemplate = ko.observable(spec[self.columnKey].case_tile_template);
         self.caseTileFields = options.caseTileFields;
-        self.caseTileTemplate = ko.observable(spec[self.columnKey].case_tile_template ? "yes" : "no");
+        self.caseTileFieldsForTemplate = ko.computed(function () {
+            return self.caseTileFields[self.caseTileTemplate()];
+        });
         self.showCaseTileColumn = ko.computed(function () {
-            return self.caseTileTemplate() === "yes" && hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_TILE');
+            return self.caseTileTemplate() !== "None" && hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_TILE');
```

Comment:
Based on my comment before I think it would be best to make this a constant and use it both for setting the value and comparing against it. Also I am not sure how this would work with translations. And do we think we need translations for the template names?

Reply:
I just meant the constant, but I'm eager to look at the translation support, that's a good idea.